etna/datasets/tsdataset.py

```diff
@@ -221,6 +221,16 @@ def segments(self) -> List[str]:
         """Get list of all segments in dataset."""
         return self.df.columns.get_level_values("segment").unique().tolist()
 
+    @property
+    def regressors(self) -> Dict[str, List[str]]:
```

Comment:
we need compute regressors only twice: in `__init__` (regressors from df_exog) and in `transform` (regressors from transform outputs) methods, don't we? 🤔